cocos/renderer/core/gfx/GFXDevice.h

```diff
@@ -67,8 +67,8 @@ class CC_CORE_API GFXDevice : public Object {
   CC_INLINE bool hasFeature(GFXFeature feature) const { return _features[static_cast<uint8_t>(feature)]; }
   CC_INLINE void defineMacro(const String& macro, const String& value) { _macros[macro] = value; }
   CC_INLINE void setReverseCW(bool reverseCW) { _reverseCW = reverseCW; }
-    CC_INLINE float getMinClipZ() const { return _minClipZ; }
-    CC_INLINE int getProjectionSignY() const { return _projectionSignY; }
+  CC_INLINE float getMinClipZ() const { return _minClipZ; }
+  CC_INLINE float getProjectionSignY() const { return _projectionSignY; }
```

Comment:
Is there any reason to use float?

Reply:
these factors are most commonly seen as parameters to projection matrices or vertex datas, both of which are of floating point type